modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -3,13 +3,15 @@
 -->
 <template>
   <div class="apos-schema">
-    <div v-for="field in currentSchema" :key="field.name">
+    <div v-for="field in schema" :key="field.name">
```

Comment:
Previously, only the fields that needed to appear on the current tab were validated. Now, every field is used for validation. So need to hide fields belonging to other tabs through `displayComponent` hereunder.

Reply:
Why is every field in the doc needed for validation?

Reply:
The fielgs belonging to other tabs are not displayed but still need to be validated before save.